fabric/fabric19/build.gradle

```
@@ -0,0 +1,45 @@
+plugins {
+    id 'fabric-loom' version '0.12-SNAPSHOT'
+}
+
+sourceSets {
+    main {
+        java.srcDirs = ['src']
+        resources.srcDirs = ['resources']
+    }
+}
+
+java.toolchain.languageVersion = JavaLanguageVersion.of(17)
+
+archivesBaseName = 'NegativityFabric-19'
+
```

```suggestion

loom {
    serverOnlyMinecraftJar()
}

```

Autrement j'ai une erreur à cause d'une classe multi-version de LWJGL qui requiert Java 19, et puis on n'a pas besoin des classes client pour un mod serveur uniquement.

De même pour la 1.19.2.